bots/pr/src/main/java/org/openjdk/skara/bots/pr/AdditionalConfiguration.java

```diff
@@ -31,14 +31,14 @@
 import java.util.*;
 
 public class AdditionalConfiguration {
-    static List<String> get(ReadOnlyRepository repository, Hash hash, HostUser botUser, List<Comment> comments, boolean reviewMerge) throws IOException {
+    static List<String> get(Optional<JCheckConfiguration> conf, HostUser botUser, List<Comment> comments, boolean reviewMerge) throws IOException {
```

Comment:
Using Optional as parameter isn't recommended, especially since we throw if it's empty. Is it truly optional here? I would think we should just `.orElseThrow()` at the parse call.

Reply:
Thanks, removed the use of `Optional` in the latest version.